action/action_test.go

```diff
@@ -1051,3 +1050,84 @@ func TestSaveAction(t *testing.T) {
 		assert.Error(t, err, "the output should NOT be persisted")
 	})
 }
+
+func Test_expandCredentials(t *testing.T) {
```

Comment:
nit: is there a reason for the different naming of this test?
(i.e. why not `TestExpandCredentials`?)

Reply:
No particular reason for the test name so updated to `TestExpandCredentials` 👍